koku/masu/database/report_db_accessor_base.py

```diff
@@ -114,12 +114,19 @@ def _execute_raw_sql_query(self, table, sql, bind_params=None, operation="UPDATE
     def _execute_trino_raw_sql_query(self, sql, *, sql_params=None, context=None, log_ref=None, attempts_left=0):
         """Execute a single trino query returning only the fetchall results"""
         results, _ = self._execute_trino_raw_sql_query_with_description(
-            sql, sql_params=sql_params, context=context, log_ref=log_ref, attempts_left=attempts_left
+            sql, sql_params=sql_params, context=context, log_ref=log_ref
         )
         return results
 
+    @retry(retry_on=(Exception,), context_extractor=extract_context_from_sql_params)
```

Comment:
The defaults can be used here. I don't think we need to specify a context extractor. Just use `ReportDBAccessorBase.extract_context_from_sql_params` directly in the decorator for now. 
```suggestion
    @retry
```

Reply:
The issue here is that we get a circular import. `ReportDBAccessorBase` imports the decorator, and the decorator imports `ReportDBAccessorBase`